blog-post.html

```diff
@@ -1,12 +1,150 @@
+    <section class="line-break">
+    </section><br>
+
+    <centre>
+      <h3>This is a <b>Blog Post</b> Only Layout</h3>
+    </centre>
+
+    <!----Main Article Blog post-->
+    <section id="main">
+
+      <div class="blog_post_page" id="main-content">
+        
+          <!.... creating container for animation...>
+          
+
+        <img src='\images\thali.jpg' alt="Healthy" width="630" height="250" />
+        <br>
+        <h3> And Yeah, that's basically it</h3>
+        <p>Lorem ipsum dolor sit amet consectetur, adipisicing
+          elit. Vitae, ipsa voluptates doloremque saepe
+          necessitatibus, quas deleniti, consequatur temporibus
+          atae atque facere alias suscipit? Atque quos nisi quod
+          ut eos quam repellat dolor! Totam rem, itaque enim
+          accusamus dolore autem asperiores quidem. Dignissimos,
+          quisquam illum repellat cum officiis veniam unde
+          similique labore aut! Porro, quae nam eligendi molestiae
+          ab minus delectus soluta vitae, fugiat quo asperiores nulla
+          reiciendis voluptates suscipit quas temporibus ea! Voluptas,
+          natus aperiam labore soluta veritatis laboriosam, officia
+          dolorum molestiae sapiente facere, id porro cupiditate
+          ipsum dignissimos mollitia nobis possimus? Laboriosam atque
+          quasi, delectus dolorem et illo beatae.</p>
+
+        <!-------My own video from Youtube :)-->
+        <h3> Check out this video to make your own Vitamin-C serum at home</h3>
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/5BmhfilFxI0" frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen></iframe>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing
+          elit. Totam, consectetur repudiandae sint aliquam
+          optio doloribus minima officia, iusto vel quo sit
+          perferendis nisi! Libero hic culpa maxime odit
+          consectetur quo.</p>
+        <h3>Ligula Suspcipt Fusce Veroeros</h3>
+        <p>Lorem ipsum dolor sit amet consectetur
+          adipisicing elit. Placeat molestias aut
+          illum aliquam quis velit enim ad totam
+          dolorum distinctio!</p>
+
+        <p>Lorem, ipsum dolor sit amet consectetur
+          adipisicing elit. Molestiae possimus
+          repellendus molestias maxime! Voluptate
+          fugiat cum, eveniet dolorum porro quod
+          dicta deserunt fuga aliquam? Laborum molestias
+          molestiae vitae aperiam tenetur omnis aspernatur
+          beatae, quae, doloremque, nostrum ex architecto
+          illo? Provident facere quasi aliquid, culpa
+          reprehenderit eum pariatur recusandae! Sequi,
+          officiis!</p>
+
+          <div id="anim-container">
+              <div id='ayurveda_love' class='food_love'>
+            <p>Love for Food!</p>
+              </div>
+              </div>
+      </div>

```

Comment:
Awesome use of animations! With a little more attention you can fix the bug where the text exits the box and just bounces back